charts/promtail/values.yaml

```diff
@@ -100,7 +100,7 @@ image:
   # -- Docker image repository
   repository: grafana/promtail
   # -- Overrides the image tag whose default is the chart's appVersion
-  tag: null
+  tag: ""
```

Comment:
With `null`, the linter complains about a type mismatch: https://github.com/grafana/helm-charts/actions/runs/9773179026/job/26978893209?pr=3200